src/renderer/components/Login.js

```diff
@@ -19,14 +20,25 @@ class Login extends React.Component {
         showAdvanced: false,
         showPasswordMail: false,
         showPasswordSend: false
-      }
+      },
+      progress: 0
     }
+    this._updateProgress = this._updateProgress.bind(this)
+    ipcRenderer.on('DC_EVENT_CONFIGURE_PROGRESS', this._updateProgress)
```

Comment:
To have a consistency, add the listener in the componentDidMount method, Besides that I think it's fine to use an inline function for a one liner.